### PR TITLE
stataUSDC for Base Mainnet

### PR DESCRIPTION
**Pull request by noahlitvin**

### CODE REVIEW

File: package-lock.json
Comment (noisekit) [inline package-lock.json:1]: Delete package lock

File: tomls/omnibus-base-mainnet-andromeda/oracles/statausdc-usdc.toml
Comment (noisekit) [inline tomls/omnibus-base-mainnet-andromeda/oracles/statausdc-usdc.toml:3]: missing target

File: tomls/omnibus-base-mainnet-andromeda/spot/statausdc.toml
Comment (noisekit) [inline tomls/omnibus-base-mainnet-andromeda/spot/statausdc.toml:23]: original token symbol `stataBasUSDC` but I am not 100% sure if I want to rename everything added here to match that... As sepolia already deployed this will not be ideal to have non-matching deployments. 
I would definitely use exact symbol name everywhere when creating all these configs from scratch though.

File: tomls/omnibus-base-mainnet-andromeda/spot/statausdc.toml
Comment (noisekit) [inline tomls/omnibus-base-mainnet-andromeda/spot/statausdc.toml:19]: We need an e2e test for this spot market. Especially considering the use of a special oracle.

File: tomls/omnibus-base-mainnet-andromeda/spot/statausdc.toml
Comment (noahlitvin) [reply tomls/omnibus-base-mainnet-andromeda/spot/statausdc.toml:23]: Yea, this was an oversight on my part, though counterpoint is if stataArbUSDC ended up on base somehow, we'd probably just use this same spot market to wrap it too.

File: tomls/omnibus-base-mainnet-andromeda/oracles/statausdc-usdc.toml
Comment (noahlitvin) [reply tomls/omnibus-base-mainnet-andromeda/oracles/statausdc-usdc.toml]: Looks like its resolved, but maybe we want to make the target `synthetix-mock-tokens` ?

File: tomls/omnibus-base-mainnet-andromeda/spot/statausdc.toml
Comment (noahlitvin) [reply tomls/omnibus-base-mainnet-andromeda/spot/statausdc.toml:19]: Added a draft test. Unsure how best to simulate the `aUSDC` rebase.

Comment (noisekit) [reply tomls/omnibus-base-mainnet-andromeda/spot/statausdc.toml:19]: Looks perfectly good, e2e here are more of a smoke tests to ensure "normal" usage scenario is working. Any special cases, reverts, etc are meant to be covered by integration/unit tests in synthetix-v3. 

Thanks for taking the time and getting the test set up and running.

File: tomls/omnibus-base-mainnet-andromeda/spot/statausdc.toml
Comment (noisekit) [reply tomls/omnibus-base-mainnet-andromeda/spot/statausdc.toml:23]: Thanks, no dramas

**Review by noisekit — APPROVED** [review]
Looking good, thanks for all the updates

**Review by noisekit — CHANGES_REQUESTED** [review]
just a temporary block until we release previous version, will merge after

**Review by noisekit — APPROVED** [review]

**Pull request merged**
